solc/CommandLineParser.cpp

```diff
@@ -967,6 +967,11 @@ General Information)").c_str(),
 		m_options.optimizer.enabled = (m_args.count(g_strOptimize) > 0);
 		m_options.optimizer.noOptimizeYul = (m_args.count(g_strNoOptimizeYul) > 0);
 
+		if (m_args.count(g_strOptimizeRuns))
+		{
+			m_options.optimizer.expectedExecutionsPerDeployment = m_args[g_strOptimizeRuns].as<unsigned>();
+		}
```

Comment:
```suggestion
			m_options.optimizer.expectedExecutionsPerDeployment = m_args[g_strOptimizeRuns].as<unsigned>();
```
(we only use curly braces if they are needed)

Reply:
I caught that just a few minutes ago, ha!